README.md

```diff
@@ -1,125 +1,175 @@
-# react-native-toast-message
+# react-native-toast-hook
 
-[![npm version](https://img.shields.io/npm/v/react-native-toast-message)](https://www.npmjs.com/package/react-native-toast-message)
-[![npm downloads](https://img.shields.io/npm/dw/react-native-toast-message)](https://www.npmjs.com/package/react-native-toast-message)
-[![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg)](https://github.com/prettier/prettier)
+[![npm version](https://img.shields.io/npm/v/@stratuscode/react-native-toast-hook)](https://www.npmjs.com/package/@stratuscode/react-native-toast-hook)
+[![npm downloads](https://img.shields.io/npm/dw/@stratuscode/react-native-toast-hook)](https://www.npmjs.com/package/@stratuscode/react-native-toast-hook)
 
-An animated toast message component for React Native that can be called imperatively.
+An animated toast message component for React Native using context API and hooks written in typescript.
 
 ## Install
 
 ```
-yarn add react-native-toast-message
+npm install --save react-native-toast-hook
 ```
 
 ![ToastSuccess](success-toast.gif)
 
 ## Usage
 
-Render the `Toast` component in your app entry file (along with everything that might be rendered there) and set a ref to it.
+Render the `ToastProvider` component in your app entry file.
 
 ```js
 // App.jsx
-import React from 'react';
-import Toast from 'react-native-toast-message';
+import React from 'react'
```

Comment:
This should be double quotes to be inline with the prettier config

Reply:
Somehow we ended up with different prettier configs. The one I've been using prefers single quotes.